docs/production-deployment/cloud/api-keys.mdx

```diff
@@ -197,55 +200,102 @@ tcld apikey disable --id <api-key-id>
 tcld apikey enable --id <api-key-id>
 ```
 
-## API Keys for Client Connections
+## Managing API Keys for Service Accounts {#serviceaccount-api-keys}
 
-Temporal Cloud supports secure programmatic access through API Key Authentication.
+Global Administrators have the ability to manage API Keys for any Service Account in their Temporal Cloud Account.
+The process to manage API Keys for Service Accounts is very similar to the process of managing API Keys for Users with one difference: Unlike User API Keys, Global Admins are able to generate API Keys for Service Accounts.
+Disabling/enabling, deleting, and viewing API Keys for Service Accounts follows the same process as API Keys for Users.
```

Comment:
These two lines (206 and 207) have parallelism between the process for Users.  Does knowing that the process is similar to "user" roles really help here?